0.X/Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM alpine:3.16
 
 # This is the release of Vault to pull in.
-ARG VAULT_VERSION=1.9.8
+ARG VAULT_VERSION=1.9.9
```

Comment:
just for history
as per twistlock scan report - 
go version 1.17.13 has 1 vulnerability
which is Fixed in: 1.19.1, 1.18.6
https://twistlock.coupahost.com:8083/#!/monitor/vulnerabilities/images/ci?search=sha256:c99567af21236372d9f7caefa215059a8e40d84d55b97b2a17adb1978774c5a6

but Vault does not have any stable release with these go versions.